packages/vite-plugin/src/node/contentScripts.ts

```diff
@@ -84,6 +86,14 @@ export function createDevLoader({
     .replace(/__TIMESTAMP__/g, JSON.stringify(Date.now()))
 }
 
-export function createProLoader({ fileName }: { fileName: string }): string {
+export function createProLoader(params: {
+  fileName: string
+  world?: 'ISOLATED' | 'MAIN'
+}): string {
+  const { fileName, world = 'ISOLATED' } = params || {}
```

Comment:
i dont get it - this function will never get parameter `{ world: 'MAIN'}`. The only usage this code does not include `world` parameter

https://github.com/crxjs/chrome-extension-tools/blob/a4982e362b15f99b7ce6d10a2348d0deb555ad90/packages/vite-plugin/src/node/plugin-contentScripts.ts#L151